index.md

```diff
@@ -1,144 +1,51 @@
-- ![area](https://img.shields.io/badge/Simulation-darkblue)
-![technique](https://img.shields.io/badge/ABM-lightgrey)
-![datamodality](https://img.shields.io/badge/TabularData-green) &nbsp; [Building an NHS Agent Based Model in HashAi](https://nhsx.github.io/nhsx-internship-projects/agent-based-model-hash-ai/)
-- ![area](https://img.shields.io/badge/NLP-yellow)
-![technique](https://img.shields.io/badge/Curation-lightgrey)
-![datamodality](https://img.shields.io/badge/TextData-9cf) &nbsp; [NHS Language Corpus](https://nhsx.github.io/nhsx-internship-projects/nhs-language-corpus/)
-- ![area](https://img.shields.io/badge/Synthetic-brightgreen)
-![technique](https://img.shields.io/badge/Simulation-lightgrey)
-![datamodality](https://img.shields.io/badge/TabularData-green) &nbsp; [Synthetic Data Exploration - Longitudinal](https://nhsx.github.io/nhsx-internship-projects/synthetic-data-exploration-longitudinal/)
-- ![area](https://img.shields.io/badge/Synthetic-brightgreen)
-![technique](https://img.shields.io/badge/BayesianNetworks-lightgrey)
-![datamodality](https://img.shields.io/badge/TabularData-green) &nbsp; [Synthetic Data Exploration - Probablistic Graphical Models](https://nhsx.github.io/nhsx-internship-projects/synthetic-data-exploration-probablistic-graphical-models/)
-
-#### Archived Projects
-
-- ![area](https://img.shields.io/badge/Geospatial-blue) &nbsp; [Granular Mapping](https://nhsx.github.io/nhsx-internship-projects/granular-mapping/)
-- ![technique](https://img.shields.io/badge/ImageSegmentation-lightgrey)
-![datamodality](https://img.shields.io/badge/Images-ff69b4) &nbsp;  [NHS GreenSpace](https://nhsx.github.io/nhsx-internship-projects/nhs-greenspace/)
-- ![area](https://img.shields.io/badge/NLP-yellow)
-![technique](https://img.shields.io/badge/SematicSearch-lightgrey)
-![datamodality](https://img.shields.io/badge/MultiModalData-blueviolet) &nbsp; [NHS Semantic Search](https://nhsx.github.io/nhsx-internship-projects/nhs-semantic-search/)
-
----
-More information on the internship scheme can be found on the [NHS England internship scheme for innovation and analytics in health](https://www.nhsx.nhs.uk/key-tools-and-info/nhsx-analytics-unit/nhsx-internship-scheme-innovation-and-analytics-health/) website.
+# {{page.title}}
+
+These internships are aimed at current PhD students looking for an industrial placement of around 5-months with the right to work in the UK.  The projects are focussed on innovation in particular around getting the most value out of NHS data.  They often have a focus on emerging data science techniques and so we advertise mainly to data science programmes however previous interns have included disciplines such as clinical, mathematics, computer science and bioinformatics which have added huge value through the range of approaches and knowledge.
```

Comment:
`The projects are focussed on innovation in particular around getting the most value out of NHS data.` -> `The projects are focussed on innovation, in particular around getting the most value out of NHS data.`

Reply:
`however previous interns have included disciplines such as clinical,` -> `however previous interns have come from other disciplines such as clinical,`